addon/controllers/i-i-s-caseberry-logging-objects-application-log-l.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * @module ember-flexberry
+ */
+import ListFormController from 'ember-flexberry/controllers/list-form';
```

Comment:
Short reexport syntax is much better: 
`export { default } from 'ember-flexberry/controllers/list-form';`